src/driver.ts

```diff
@@ -32,18 +33,27 @@ export class HTTPDriver implements Driver {
   }
 
   public getDatabase(): Database {
-    return new DatabaseHttp(this.openApiClient.database);
+    return new DatabaseHttp(
+      this.openApiClient.database,
+      this.driverConfig.projectName,
+      this.driverConfig.branch
+    );
   }
 
-  public async initialize() {
-    this.accessToken = await new HttpAuthorization({
-      clientId: this.driverConfig.clientId,
-      clientSecret: this.driverConfig.clientSecret,
-    }).getAuthorizationHeader();
+  public async initialize(authEnabled?: boolean) {
```

Comment:
The access token generated is only valid for 24 hours. How do we handle the token refresh?